src/components/redux/auth/auth_action_types.js

```diff
@@ -0,0 +1,2 @@
+export const login = () => ({ type: "LOGIN" });
+export const logout = () => ({ type: "LOGOUT" });
```

Comment:
I suggest creating a simulated server-like response.
```suggestion
export const login = () => ({ type: "LOGIN", payload: { success: true } });
export const logout = () => ({ type: "LOGOUT", payload: {success: false} });
```